Library/ViewModels/ShareViewModel.swift

```diff
@@ -42,17 +41,11 @@ public protocol ShareViewModelInputs {
   /// Call from the UIActivityViewController's completion handler.
   func shareActivityCompletion(with data: ShareActivityCompletionData)
 
-  /// Call from the SLComposeViewController's completion handler
-  func shareComposeCompletion(result: SLComposeViewControllerResult)
```

Comment:
It also seems that `func facebookButtonTapped` and `func twitterButtonTapped` are unused

Reply:
Ah thanks!